contracts/polling/PollCreator.sol

```diff
@@ -12,26 +17,28 @@ contract PollCreator {
     uint256 public constant POLL_PERIOD = 10 * 5760;
     uint256 public constant POLL_CREATION_COST = 100 * 1 ether;
 
-    ILivepeerToken public token;
+    IBondingManager public bondingManager;
 
     event PollCreated(address indexed poll, bytes proposal, uint256 endBlock, uint256 quorum, uint256 quota);
 
-    constructor(address _tokenAddr) public {
-        token = ILivepeerToken(_tokenAddr);
+    constructor(address _bondingManagerAddr) public {
+        bondingManager = IBondingManager(_bondingManagerAddr);
     }
 
     /**
-     * @dev Create a poll by burning POLL_CREATION_COST LPT.
-     *      Reverts if this contract's LPT allowance for the sender < POLL_CREATION_COST.
+     * @notice Create a poll if caller has POLL_CREATION_COST LPT stake (own stake or stake delegated to it).
      * @param _proposal The IPFS multihash for the proposal.
      */
     function createPoll(bytes calldata _proposal) external {
         uint256 endBlock = block.number + POLL_PERIOD;
         Poll poll = new Poll(endBlock);
 
-        require(token.transferFrom(msg.sender, address(this), POLL_CREATION_COST), "LivepeerToken transferFrom failed");
-
-        token.burn(POLL_CREATION_COST);
+        require(
+            // pendingStake() ignores the second arg
+            bondingManager.pendingStake(msg.sender, 0) >= POLL_CREATION_COST ||
+                bondingManager.transcoderTotalStake(msg.sender) >= POLL_CREATION_COST,
+            "PollCreator#createPoll: INSUFFICIENT_STAKE"
+        );
```

Comment:
should move this require statement before contract deployment to considerably reduce consumed gas in case of a failure/revert.

Reply:
Fixed in c3806a6